repos/fdbt-site/tsconfig.json

```diff
@@ -24,12 +24,7 @@
     "rootDir": "./",
     "jsx": "preserve",
     "sourceMap": true,
-    "baseUrl": ".",
-    "paths": {
-      "src/*": [
-        "./src/*"
-      ]
-    }
+    "baseUrl": "."
```

Comment:
The stuff that is removed was added by @RobCatton for his IDE, am I right? Will this break things for VS Code users?

Reply:
This will actually help because this causes imports to be added starting with 'src/' rather than '../../src/'